baseband/guppi/header.py

```diff
@@ -336,7 +336,7 @@ def offset(self):
 
     @offset.setter
     def offset(self, offset):
-        self['PKTIDX'] = int(round((offset.to_value(u.s) / self['TBIN'] /
```

Comment:
Why this change? `to_value(u.s)` is not only allowed but quite a bit faster than first creating this value, turning it in a quantity, and then back into an array.

Reply:
I think (since I did it in the tests) that it's natural to try to pass a `TimeDelta` rather than a quantity to `offset`, which doesn't have the `to_value` attribute.

Reply:
OK, fair enough! That does remind me of https://github.com/astropy/astropy/issues/6148